projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/radio-button-docs/radio-button-docs.component.html

```diff
@@ -74,6 +74,47 @@ <h4 class="go-heading-4 go-heading--underlined">Code</h4>
         </div>
       </div>
 
+      <div class="go-column go-column--100">
+        <p class="go-body-copy">
+          Additionally, there are two more bindings: <code class="code-block--inline">enableFieldset</code> and
+          <code class="code-block--inline">enableLegend</code>. You can use these to show/ hide the fieldset and (or)
+          the legend, respectively. By default, both are set to <code class="code-block--inline">true</code>.
+        </p>
+        <p class="go-body-copy">
+          Refer to the example below:
+        </p>
```

Comment:
I think we can probably remove this, the example is directly below it. The rest of our docs follow this format as well, so it's consistent.